android-components/plugins/dependencies/settings.gradle

```diff
@@ -0,0 +1,3 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Out of curiosity: what does this empty file do for Gradle?

Reply:
@JohanLorenzo We had Android Studio complain during "clean" if the file wasn't present. It shouldn't do anything, and on the command line it didn't matter at all, but it didn't seem worth debugging further.